plutus-tx-plugin/test/CallTrace/9.6/func05.eval.golden

```diff
@@ -7,4 +7,4 @@ Trace:
 -> $cmyClassFunc (test/CallTrace/Spec.hs:99:3-99:13)
 -> functionFromOtherModule (test/CallTrace/Spec.hs:93:1-93:23)
 -> $
--> errorWhenTrue (test/CallTrace/OtherModule.hs:9:1-9:13)
\ No newline at end of file
+-> errorWhenTrue
```

Comment:
I don't know what this is. I've rebased on `master` and currently rerunning the tests.